__tests__/integration/app.test.js

```diff
@@ -51,14 +51,69 @@ describe("GET /api/topics", () => {
 				});
 			});
 	});
-	describe("Error handling", () => {
-		test("400: /api/notavalidpath", () => {
-			return request(app)
-				.get("/api/notavalidpath")
-				.expect(404)
-				.then(({ body }) => {
-					expect(body.msg).toBe("path does not exist");
+});
+
+describe("GET /api/articles/:article_id", () => {
+	test("200: responds with a correct article object", () => {
+		return request(app)
+			.get("/api/articles/1")
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.article).toEqual({
+					article_id: 1,
+					title: "Living in the shadow of a great man",
+					topic: "mitch",
+					author: "butter_bridge",
+					body: "I find this existence challenging",
+					created_at: "2020-07-09T20:11:00.000Z",
+					votes: 100,
+					article_img_url:
+						"https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
 				});
-		});
+			});
+	});
+	test("an article object should have properties: author, title, article_id, body, topic, created_at, votes, article_img_url", () => {
+		return request(app)
+			.get("/api/articles/3")
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.article).toMatchObject({
+					author: expect.any(String),
+					title: expect.any(String),
+					article_id: 3,
+					body: expect.any(String),
+					topic: expect.any(String),
+					created_at: expect.any(String),
+					votes: expect.any(Number),
+					article_img_url: expect.any(String),
+				});
+			});
+	});
+});
+
+describe("Error handling", () => {
+	test("404: /api/notavalidpath", () => {
+		return request(app)
+			.get("/api/notavalidpath")
+			.expect(404)
+			.then(({ body }) => {
+				expect(body.msg).toBe("path does not exist");
+			});
+	});
+	test("GET:400 responds with an appropriate error message when given an invalid id", () => {
```

Comment:
As these tests are not generic error handling, I would put them with their relevant endpoints